dev/CMakePresetsMamba.json

```diff
@@ -23,7 +23,8 @@
     {
       "cacheVariables": {
         "BUILD_MICROMAMBA": "ON",
-        "BUILD_STATIC": "ON"
+        "BUILD_STATIC": "ON",
+        "ENABLE_WIN32_XMLLITE": "ON"
```

Comment:
Do we still need it in the preset if it is always set in the CMakeLists.txt file?

Reply:
The CI was passing withtout that change, I didnt try it yet after removing the explicit link to `xmllite.lib` which I find weird. It makes sense as a system-provided library, but libarchive already has it's own way of using it ([see there](https://github.com/libarchive/libarchive/blob/cc97338239c5b1c0e0ffcb323d8cc4fd5929bf54/CMakeLists.txt#L1267)) but  enabling it in our mamba cmake file is not enough to make this work apparently.
Not sure what's best.